### DISCUSSION TITLE
Steps to use prepISO  ?

### DISCUSSION THREAD

**Discussion by mailliwal** [aldostools/webMAN-MOD#1024] [Q&A]
Hi,

After installed prepISO, may I know need to execute every time ?

1) Enable HEN
2) Connect and Power-on USB HDD (MBR and exFAT)
3) Execute prepISO

Since I have tried sometime not detected USB HDD or black screen after executed.

Thanks

**Comment by aldostools** [comment]
prepISO is only needed when you add or delete games from the external HDD or if you change the HDD from USB port.

It is important that you check the option "Disable content scan on startup" and SAVE the settings.

The black screen is normal in prepISO if there is a large number of games. The application doesn't have a GUI to speedup the scanning.

After the scanning is completed, the application returns to XMB.

Make sure that you have installed the latest prepISO 1.33. The version 1.32 had a bug with FakeISO stored in exFAT.

If you have issues with exFAT, you can try formatting your HDD to NTFS which uses a different library and seems to be more stable.

**Reply by mailliwal** [reply]
- Added means copy to internal HDD ?
- After enabled HEN, sometime games under Playstation3 folder will only showed internal HDD but no external USB HDD
- Where can "Disable content scan on startup" ?

Thanks

**Reply by aldostools** [reply]
* "when add or delete games from the **external** HDD" means when copied to external HDD or removed
* In [/setup.ps3](https://github.com/aldostools/webMAN-MOD/wiki/Setup-Options#scan-for-content) uncheck /ntfs
![image](https://github.com/aldostools/webMAN-MOD/assets/974897/86f4f6ec-2c8c-4b79-a881-0d702791d815)
* In [/setup.ps3](https://github.com/aldostools/webMAN-MOD/wiki/Setup-Options). Remember to SAVE the settings after change the options. 
![image](https://github.com/aldostools/webMAN-MOD/assets/974897/61e9ff84-9483-41e0-85cc-bd78214deb7c)
